### ISSUE TITLE
Update API refs

### ISSUE THREAD

**Issue by OculusMode** [nhnent/tui.image-editor#42]
## Version
3.2.0
## Current Behavior
1. Please check the text bar in api example. backspace doesnt seem to work. and a bit strange behaviour. it is working in 3.1.0 though.
2. Please Update Api refs to how to change brightness as well as how can i apply filters with more or less values.
Thanks.

**Comment by OculusMode** [comment]
Edit: Can i edit saturation and contrast?
Thanks in advance :)

**Comment by jinwoo-kim-nhn** [comment]
Hello @OculusMode 

1. I figured out what was the problem. I will solve this in a very short time.

2. We plan to augment the detailed documentation related to filter. Just wait a little.

3. There is a 'filter> Grayscale' feature with respect to saturation, but the specific detail editing is not yet supported.
But please wait a moment because we have the will to improve.

**Comment by OculusMode** [comment]
Thank you for reply and support.
I found brightness by value in your basic demo code which can be like

`imageEditor.applyFilter("brightness", { brightness: parseInt(val,10 )})`

can you just tell me what it would be expecting for another paras for grayscale?

`imageEditor.applyFilter("grayscale", { ???: parseInt(val,10 )})`

**Comment by ivanyiistvan** [comment]
Hello @jinwoo-kim-nhn!

Are there any changes about the saturation feature?

Thanks!

**Comment by junghwan-park** [comment]
@OculusMode 
You can pass only filter name.`imageEditor.applyFilter('grayscale');`
Additional options are not needed.

**Comment by Alecyrus** [comment]
@junghwan-park there is no possible to change the value of `filter` effect like css, `filter: grayscale(value)` ??

**Comment by junghwan-park** [comment]
@Alecyrus 
There is no options for default values yet. ex) filter, text size and etc
But, I'm planning to add customizing options for default UI. :)

**Comment by Alecyrus** [comment]
@junghwan-park Thank you for this nice work that inspires me.  And it is important to note that, filter is a  good feature but built-in filter is not practical. I am really looking forward there is some libraries like GPUImages on JS.  but now, I am  trying to use 3D LUTS to process image on NodeJS which could bring me many amazing effects.

**Comment by junghwan-park** [comment]
@Alecyrus 
I agree with you what you said. Built-in filters are not practical for users from now. But I'm considering to add UI customization API(including add/remove custom menu and set custom filters). Through that API, users could add custom filter like `Analog Film` I guess.

If you have an idea of adding custom filter feature or image processing extensions, please let us know with posting a new issue. Thanks :)
